site/src/components/WorkspaceActions/ActionCtas.tsx

```diff
@@ -86,8 +124,26 @@ const useStyles = makeStyles((theme) => ({
   actionButton: {
     // Set fixed width for the action buttons so they will not change the size
     // during the transitions
-    width: theme.spacing(16),
+    width: "160px",
     border: "none",
     borderRadius: `${theme.shape.borderRadius}px 0px 0px ${theme.shape.borderRadius}px`,
   },
+  cancelButton: {
+    "&.MuiButton-root": {
+      padding: "0px 0px !important",
+      border: "none",
+      borderLeft: `1px solid ${theme.palette.divider}`,
+      borderRadius: `0px ${theme.shape.borderRadius}px ${theme.shape.borderRadius}px 0px`,
+      width: "63px", // matching dropdown button so button grouping doesn't grow in size
```

Comment:
Maybe we can add some UI snapshots with the dropdown open to verify these changes?